js/adapt-assessmentArticleModel.js

```diff
@@ -388,15 +384,64 @@ define([
             this.set("_isPass", isPass);
         },
 
-        _isAttemptsLeft: function() {
+        _getMarkingSettings: function() {
+            var markingSettings = {};
+
+            if (this._shouldSuppressMarking()) {
+                markingSettings = {
+                    _canShowMarking: false,
+                    _canShowModelAnswer: false
+                };
+            } else {
+                var questionConfig = this.getConfig()._questions;
+
+                if (questionConfig.hasOwnProperty('_canShowModelAnswer')) {
+                    markingSettings._canShowModelAnswer = questionConfig._canShowModelAnswer;
+                }
+
+                if (questionConfig.hasOwnProperty('_canShowMarking')) {
+                    markingSettings._canShowMarking = questionConfig._canShowMarking;
+                }
+            }
+
+            return markingSettings;
+        },
+
+        _overrideMarkingSettings: function() {
+            var newMarkingSettings = this._getMarkingSettings();
+            for (var i = 0, l = this._currentQuestionComponents.length; i < l; i++) {
+                this._currentQuestionComponents[i].set(newMarkingSettings, {
+                    pluginName: "_assessment"
+                });
+            }
+        },
+
+        _refreshQuestions: function() {
+            for (var a = 0, b = this._currentQuestionComponents.length; a < b; a++) {
+                var question = this._currentQuestionComponents[a];
+                question.refresh();
```

Comment:
the pr for this interface is really not right. so i'm revoking my +1 until that's fixed. please see comments at: 

https://github.com/adaptlearning/adapt_framework/pull/1360#discussion_r104629601